application/sitebuilder/build.py

```diff
@@ -493,3 +495,7 @@ def cleanup_filename(filename):
 
 def get_static_dir():
     return "static"
+
+
+def _stringify_timestamp(_timestamp):
+    return _timestamp.strftime(BUILD_TIMESTAMP_FORMAT)
```

Comment:
Given that this is the only place using the `BUILD_TIMESTAMP_FORMAT` constant, it could be included here as a string literal instead?